db/seeds.rb

```diff
@@ -29,7 +29,7 @@ def generate_property_types
   end
 
   def generate_properties
-    user_ids = User.all.pluck(:id)
+    user_ids = User.all.pluck(:id).take(100)
```

Comment:
you don't need that "#all" I just double checked on my rails console and you can just do 

User.pluck(:id).take(100)